packages/frontend/src/components/Game/GameBoard/index.test.jsx

```diff
@@ -68,4 +68,49 @@ describe('GameBoard component', () => {
     expect(tile.photo(tiles[1].photo.alt).query()).not.toBeInTheDocument();
     expect(tile.photo(tiles[5].photo.alt).query()).not.toBeInTheDocument();
   });
+
+  it('will not flip a tile to face down with second click', async () => {
+    const tiles = createTilesFromPhotos(mockPhotos, { shuffle: false });
+
+    const flippedTileState = produce(baseState.game, (draft) => {
+      draft.tiles = tiles;
+    });
+
+    const state = { ...baseState, game: flippedTileState };
+
+    const { user, screen } = setupTests(GameBoard, { state });
+
+    expect(screen.getByTestId('tile-0')).toHaveClass('faceDown');
+
+    await user.click(screen.getByTestId('tile-0'));
+
+    act(() => jest.advanceTimersByTime(3000));
+
+    await user.click(screen.getByTestId('tile-0'));
+
+    act(() => jest.advanceTimersByTime(1000));
+
+    expect(screen.getByTestId('tile-0')).toHaveClass('faceUp');
```

Comment:
This test has some things that might not be necessary. To me it seems like this test should:
- Select a tile
- Assert that the tile is face down
- Click the tile
- Assert that the tile is face up
- Click the tile again
- Assert that the tile is *still* face up

I'm not sure the timeouts are required, as the user click events are async and will wait until the document is rerendered before moving on, I believe.

Reply:
Totally. The timers make no difference here. Good catch.